.env

```diff
@@ -1032,3 +1033,21 @@ TARANTOOL_ADMIN_PORT=8002
 NATS_CLIENT_PORT=4222
 NATS_MONITORING_PORT=6222
 NATS_ROUTE_PORT=8222
+
+### SOKETI ##################################################
+SOKETI_PORT=6001
+SOKETI_METRICS_SERVER_PORT=9601
+
+WORKSPACE_INSTALL_ZMQ=false
+WORKSPACE_COMPOSER_AUTH_JSON=false
+WORKSPACE_INSTALL_XLSWRITER=false
+WORKSPACE_INSTALL_EVENT=false
+PHP_FPM_INSTALL_ZMQ=false
+PHP_FPM_INSTALL_XLSWRITER=false
+PHP_FPM_INSTALL_EVENT=false
+PHP_WORKER_INSTALL_XLSWRITER=false
+PHP_WORKER_INSTALL_EVENT=false
+WORKSPACE_WKHTMLTOPDF_VERSION=0.12.6-1
+
+POSTGIS_VERSION=latest
+POSTGIS_INSTALL_PGSQL_HTTP_FOR_POSTGIS13=false
```

Comment:
could you put these lines in same places where .env.exam ple have them,
so `diff .env .env.example` would show only changed lines (not blocks changed places)